examples/first-component/src/components/NewTodo/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from "react";
+import { createTodo } from "../TodoList/todos";
+import { Todo } from "../../types/Todo";
+
+interface NewTodoProps {
+  onAdd: (newTodo: Todo) => void;
+}
+
+const NewTodo: React.FC<NewTodoProps> = ({ onAdd }) => {
+  const [text, setText] = useState("");
+
+  const handleAddClick = () => {
+    onAdd(
+      createTodo({
+        text: text,
+        completed: false,
+      })
```

Comment:
create variable for this in order to more readable

Reply:
Fixed